universal-application-tool-0.0.1/test/services/PathTest.java

```diff
@@ -72,6 +72,32 @@ public void keyName() {
     assertThat(path.keyName()).isEqualTo("name");
   }
 
+  @Test
+  public void isList() {
+    Path path = Path.create("one.two[3]");
+    assertThat(path.isList()).isTrue();
+  }
+
+  @Test
+  public void withoutArrayIndex() {
+    Path path = Path.create("one.two[3]");
+    assertThat(path.withoutArrayIndex()).isEqualTo(Path.create("one.two"));
+  }
+
+  @Test
+  public void arrayIndex() {
+    Path path = Path.create("one.two[3]");
+    assertThat(path.arrayIndex()).isEqualTo(3);
```

Comment:
suggestion (blocking): could we add a test case for multi-digit numbers, just to prove the regex works?

Reply:
done